src/System.Memory/tests/Span/SequenceCompareTo.bool.cs

```diff
@@ -0,0 +1,149 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
```

Comment:
We also need test coverage for ReadOnlySpan.

The tests should look similar to the ones you just added here, except call SequenceCompareTo on `ReadOnlySpan<T>` instead of `Span<T>`.

It also looks like we need to port over the existing byte specific tests from Span to ReadOnlySpan (copy and modify). https://github.com/dotnet/corefx/blob/5f4e97d668677210cb41acf9f0df0e2d71382cb5/src/System.Memory/tests/Span/SequenceCompareTo.byte.cs#L12

See other ReadOnlySpanTests if you have any questions: https://github.com/dotnet/corefx/blob/5f4e97d668677210cb41acf9f0df0e2d71382cb5/src/System.Memory/tests/ReadOnlySpan/StartsWith.byte.cs#L9

If you add the ROS tests, please add the cs files to the appropriate ItemGroup - https://github.com/dotnet/corefx/blob/5f4e97d668677210cb41acf9f0df0e2d71382cb5/src/System.Memory/tests/System.Memory.Tests.csproj#L128

Reply:
My last commit addresses the ROS tests. Hope everything will be alright !